packages/arb-bridge-eth/package.json

```diff
@@ -42,6 +42,7 @@
   },
   "dependencies": {
     "@openzeppelin/contracts": "3.4.0",
+    "@openzeppelin/contracts-0.8": "npm:@openzeppelin/contracts@^4.3.1",
```

Comment:
We should probably use the upgradeable version instead
`https://github.com/OpenZeppelin/openzeppelin-contracts-upgradeable/tree/v4.3.1`

Reply:
no proxy will be used